### PR TITLE
Remove makefile change needed for macos sed

### PR DESCRIPTION
**Pull request by rossf7**
I just released v1.1.0 with the github action.

However I'm using mac sed 😬 and needed to make this change. Should we revert it or try to make it support linux and macos?

### CODE REVIEW

**Comment by piontec** [comment]
> I just released v1.1.0 with the github action.
> 
> However I'm using mac sed grimacing and needed to make this change. Should we revert it or try to make it support linux and macos?

Ugh... let's try to make it work on both.

**Comment by rossf7** [comment]
> Ugh... let's try to make it work on both.

I remember now. In the past I've done this but `--with-default-names` got removed from brew.

```
brew install gnu-sed --with-default-names
```

I'll have a go at making the makefile support both OSes tomorrow. 👍

File: Makefile.abs.mk [hunk 1]
```diff
@@ -30,8 +30,7 @@ release_ver_to_code:
 	$(call check_defined, TAG)
 	echo "build_ver = \"${TAG}\"\n" > app_build_suite/version.py
 	$(eval IMG_VER := ${TAG})
-	cp dabs.sh dabs.sh.back
-	bash -c 'sed -i "" "s/latest/$${TAG#v}/" dabs.sh'
+	bash -c 'sed -i.back "s/latest/$${TAG#v}/" dabs.sh'
```
Comment (rossf7) [inline Makefile.abs.mk:33]: `-i` supports an optional suffix to create a temp file which we were doing already.

This should allow the Makefile to support both OSes.

See https://stackoverflow.com/questions/16745988/sed-command-with-i-option-in-place-editing-works-fine-on-ubuntu-but-not-mac/44864004#44864004 for more details

**Review by ubergesundheit — APPROVED** [review]
I trust you have tested this on both GNU and BSD coreutils :)

**Review by piontec — APPROVED** [review]
LGTM!

**Comment by rossf7** [comment]
This didn't work. I've reverted my previous change and added a note to docs to use gsed on mac.

**Pull request merged**